estate/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+{
+    "name": "Real Estate",
+    "version": "0.1",
+    "category": "Administration",
+    "sequence": 100,
+    "license": "LGPL-3",
+    "author": "alup",
```

Comment:
```suggestion
    "author": "Odoo S.A.",
```
We work for Odoo S.A. :smile: